silq/instrument_interfaces/keysight/Keysight_SD_AWG_interface.py

```diff
@@ -647,6 +653,37 @@ def implement(self, interface, instrument, default_sampling_rate, threshold):
 
         return [waveform]
 
+class FrequencyRampPulseImplementation(PulseImplementation):
```

Comment:
Would be good to have a check to see that the waveform isn't some crazy high amount of points, there's always the risk with ramp pulses